app/src/src/components/views/LikedUserView.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import { Text, View, StyleSheet, Image } from 'react-native';
+// import ChatBubbleIcon from "@material-ui/icons/ChatBubble";
+import {
+  configureFonts,
+  DefaultTheme,
+  Provider as PaperProvider,
+} from 'react-native-paper';
+
+const LikedUserView = (props) => {
+  const { first_name, profile_photo } = props;
```

Comment:
We can get rid of line 11 by modify line 10 to be
`const LikedUserView = ({ first_name, profile_photo }) => {`
This helps with intellisense

Reply:
👍